edgedetector/solver/crossover/crossover_factory.py

```diff
@@ -0,0 +1,15 @@
+"""This class is responsible for creating objects of subclasses of abstract
+class Crossover"""
+
+from edgedetector.solver.crossover.random_crossover import RandomCrossover
+
+
+class CrossoverFactory:
+
+    @staticmethod
+    def create(config):
+        probability = config['probability']
+        class_ = config['class']
+        if class_ == "RandomCrossover":
```

Comment:
We would have to pass full path to module in which the class to be instantiated is implemented or move subclasses of given abstract class to one module. In my opinion current solution is more readable